_build/index.js

```diff
@@ -20,23 +20,36 @@ var _saleRoutes = require('./server/routes/sale-routes');
 
 var _saleRoutes2 = _interopRequireDefault(_saleRoutes);
 
+var _userRoutes = require('./server/routes/user-routes');
+
+var _userRoutes2 = _interopRequireDefault(_userRoutes);
+
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```

Comment:
Your code should be readable. Ensure you use es6 arrow functions here

Reply:
I think I made a mistake of pushing my build folder to the git repo.

Reply:
Because that's the transpiled es5 version